src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -78,7 +78,7 @@ std::string HttpInfo::toJson()
 
   if (custom)
   {
-    if (!this->payload.empty())
+    if (!this->includePayload)
```

Comment:
Not sure but I'd say this change needs to be reverted. Note this is a method to render part of the subscription payload (i.e. the one you get with `GET /v2/subscriptions`) and the `payload` field (if exist) has to be shown no matter the value of `includePayload`.

Reply:
Updated